contributor/01-introduction-article.md

```diff
@@ -7,61 +7,60 @@ Perhaps after a while you even had to do some extra work in your project to work
-Caring for the community involves keeping it healthy, upleveling it and its participants and advocating its needs in their organization.
+They also need to care about product quality, sustainability and project evolution from technical as well as general perspective, about reducing the barrier to making contributions for everyone, as well as about caring for their community in general.
+Caring for the community involves keeping it healthy, upleveling it and its participants, and advocating its needs in their organization.
 
 
 ### Product Owner
 The role of the [_Product Owner_][PO Introduction] has some similarity with the product owner role of your average project.
-However, there are some differences - depending on the size of the project, this role is often filled by the same person that acts as trusted committer.
-In larger projects or in teams that only partially use InnerSource as an approach to fill their needs by accepting contributions this role is likely filled by a person separate from a trusted committer.
+However, there are differences - depending on the size of the project, this role is often filled by the same person that acts as trusted committer.
+In larger projects, or in teams that only partially use InnerSource as an approach to fill their needs by accepting contributions, this role is likely filled by a person separate from a trusted committer.
 
-Your interaction with this role will likely focus on working out the fit of your contribution into the general product and its road map. 
-You might work this role to make sure that general aspects of documentation or consistency of UI/UX are kept upon merging your contribution.  
+Your interaction with product owner will likely focus on working out the fit of your contribution into the general product and its roadmap. 
+You might work with product owner to make sure that general aspects of documentation, or consistency of UI/UX, are kept upon merging your contribution.  
 
-Last but not least someone acting as product owner might have been involved in bringing the project, its benefits and community to your attention. 
+Last but not least, someone acting as product owner might have been involved in bringing the project, its benefits and community to your attention. 
 
 
-If you want to learn in more detail what these other roles are about, and we encourage you to do so, we've prepared sections about the [_Trusted Committer_][TC Introduction] and the [_Product Owner_][PO Introduction] too.
+If you want to learn in more detail what these other roles are about, and we encourage you to do so, we've prepared separate sections about the [_Trusted Committer_][TC Introduction] and the [_Product Owner_][PO Introduction].
 
 
 ## Section overview
-In the following 5 videos you will learn more in detail about the various aspects introduced before. 
+In the following 5 sections you will learn more in detail about the various aspects introduced here. 
```

Comment:
There's this notation of segment vs section that O'Reilly uses, that I always get mixed up.
Could @clobner or @rrrutledge help me out here?

Reply:
A section contains multiple segments.  This word should be `segment`.

```suggestion
In the following 5 segments you will learn more in detail about the various aspects introduced here. 
```